migrations/1_create_items_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE items;
```

Comment:
Would add an IF EXISTS to this e.g. `DROP TABLE IF EXISTS items;`
Like wise on the CREATE would do the same `CREATE TABLE IF NOT EXISTS items...`

Reply:
This caught me out in the next PR 😅  https://github.com/alexdunne/gs-onboarding/blob/623ea5cb10cf3a2f7aad2e2b30fcc7083b7d6efc/migrations/1_create_items_table.down.sql#L1